modules/cluster-logging-release-notes-5.0.9.adoc

```diff
@@ -0,0 +1,25 @@
+[id="cluster-logging-release-notes-5-0-9"]
+= OpenShift Logging 5.0.9
+
+This release includes link:https://access.redhat.com/errata/RHBA-2021:3705[RHBA-2021:3705 - Bug Fix Advisory. OpenShift Logging Bug Fix Release (5.0.9)].
```

Comment:
maybe it's just me, but the link doesn't seem to work.

Reply:
I am not sure how the part of the sentence before the link and in [] form a sentence.

Reply:
The broken link is expected. We wait for the advisory to be published before we merge the PR.
This line includes the advisory title and synopsis: `RHBA...` and `OpenShift...`. What you would recommend doing instead? I'd like to minimize the impact on other similar content.

Reply:
https://deploy-preview-36961--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-release-notes.html#cluster-logging-release-notes-5-0

Reply:
I would prefer to keep the format of these links as they are. Altering them will involve making a lot of low-value changes to other files.

Reply:
So basically you are saying that in the end the link will work? In that case I'd leave it as is!